### DISCUSSION TITLE
[German Keyboard] Change Hotkeys

### DISCUSSION THREAD

**Discussion by FeelsBadMan1** [community/community#7094] [Copilot Conversations] [answered]
Hello, i was wondering about how i am able to choose the next solution. I then found docs for it that show some hotkeys i should use. I use a german keyboard layout so most of the keys wont change anything as example i should use "alt + ]" but the "]" key cant be just added by pressing a sperate key. i would need to pess "strg + alt" and then the number 9 and only then the key would appear. The only options i can do is press tab to accept the solution, press esc to discard the suggestion and press alt + enter to open the different suggestion menu.

**Comment by busti** [comment, marked as answer]
You can change the keymap in `settings / keymap / <search for "copilot">"`  
That being said, there should probably be default bindings for different languages.

**Comment by daniel** [comment]
How do I change the hotkeys for this in VsCode? I can't find any for next/previous when I open Keyboard Shortcuts and search for copilot.
![Screenshot 2021-11-02 at 11 55 33](https://user-images.githubusercontent.com/635/139833907-3444f79a-9c53-4318-bd35-47ff0e55b0ee.png)

**Reply by daniel** [reply]
I solved it, it was documented [here](https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/configuring.md).

**Reply by Rblip** [reply]
Hi, thanks for sharing. The link seems dead. Any other fixed for German keyboards in VSCode?
